packages/nodes-base/nodes/Google/Sheet/v2/actions/sheet/appendOrUpdate.operation.ts

```diff
@@ -419,27 +419,27 @@ export async function execute(
 	if (appendData.length) {
 		const lastRow = sheetData.length + 1;
 		if (options.useAppend) {
-			await sheet.appendSheetData(
-				appendData,
+			await sheet.appendSheetData({
+				inputData: appendData,
```

Comment:
same as before could you add names for the rest of the parameters to make the code more consistent and readable?

Reply:
we had to keep it as is here since we have append and update data